datacite-jsonapi-annotator/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>datacite-rest-client-parent</artifactId>
+        <groupId>org.gbif.datacite</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>datacite-jsonapi-annotator</artifactId>
+
+    <name>DataCite JSON:API Annotator</name>
+    <description>DataCite JSON:API Annotator</description>
+
+    <dependencies>
```

Comment:
missing dependency com.sun.codemodel:codemodel
are jackson core and annotations needed?

Reply:
codemodel and mockito-inline are used. looks like false positive scanning result.
remove redundant jackson and others.